CONTRIBUTING.md

```diff
@@ -34,7 +34,7 @@ _Before_ submitting a pull request, please make sure the following is done…
 
 1.  Jest uses [Yarn](https://code.facebook.com/posts/1840075619545360) for running development scripts. If you haven't already done so, please [install yarn](https://yarnpkg.com/en/docs/install).
 
-1.  Make sure you have `python` installed (v2.7 is recommended, v3.x.x is not supported). Python is required by [node-gyp](https://github.com/nodejs/node-gyp) that is used when running `yarn install`.
+1.  Make sure you have `python` installed (v3.x is recommended, v2.7 [is not](https://www.python.org/doc/sunset-python-2/)). Python is required by [node-gyp](https://github.com/nodejs/node-gyp) that is used when running `yarn install`.
```

Comment:
2.7 is still supported by node-gyp, maybe we could just say

>Make sure you have `python` installed. Python is required by [node-gyp](https://github.com/nodejs/node-gyp) that is used when running `yarn install`.

?

Reply:
done